lambda_src/drivers/process_sentry_logger.py

```diff
@@ -0,0 +1,45 @@
+import logging
+
+import sentry_sdk
+from sentry_sdk import push_scope
+
+CONSOLE_LOGGER = logging.getLogger('console')
+GEFF_SENTRY_LOGGER = logging.getLogger('geff')
+SENTRY_DRIVER_LOGGER = logging.getLogger('sentry_driver')
+
+
+def process_row(
+    name: str,
+    history_type: str,
+    error: str,
+    ts: str,
+    history_url: str,
+):
+    """Each row is sent to Sentry via the SENTRY_DRIVER_LOGGER
+
+    Args:
+        name (str): The object name.
+        history_type (str): pipe or task or query.
+        error (str): The actual error message.
+        ts (str): The timestamp of the error.
+        history_url (str): URL of the erroring object in history.
+    """
+    CONSOLE_LOGGER.info(f'sentry_logger driver invoked.')
+
+    with push_scope() as scope:
+        scope.set_extra('history_url', history_url)
+        sentry_sdk.set_tag(
+            (
+                'PIPE_NAME'
```

Comment:
Starting to think PIPE_NAME, TASK_NAME and such is not needed here. @sfc-gh-afedorov I'll just receive these fields into a kwargs.